Changelog

```diff
@@ -1,3 +1,67 @@
+Version 3.1, 2019-09-XX
+
+  Features:
+  * Allow user attributes in policy conditions (#1645)
+  * Assign tokens and set old PIN during migration (#1619)
+
+  Enhancements:
+  * Authentication and Challenge Response:
+    * RADIUS token supports a single AccessChallenge with the remote RADIUS server (#1790)
+    * Improving Push token performance by reusing still valid access token (#1795)
+    * Improving TiQR token: It returns the remaining attemps after a wrong PIN is given (#1777)
+    * Enhance validate check logic in regards to serials and user names (#1768)
+    * User may now have several TiQR tokens at the same time (#1739)
+    * Do not increase fail counter when *checking* for an answered challenge (#1697)
+    * Allow additional token specific checks when answering challenge response (#1695)
+    * Endpoint GET /token/challenges also takes transaction_id (#1689)
+    * Push token can delay the response of /validate/check, so that there is no need 
+      to query the server to check if the push notification has been answered (#1583)
+  * User experience:
+    * Improve user experience when enrolling Yubikeys via ykpersonalize - Automatically 
+      removing whitespaces (#1735)
+    * Allow user to change the token description (#1717)
+    * Customize Web UI page title (#1624, #1243)
+    * *search_on_enter* also applies to audit log (#1493)
+    * Allow a welcome message in the Web UI if the user has not token (#1074)
+  * Management:
+    * Add script to export OTP counters (#1728)
+    * Admins can only see tokens within the realm they are allowed to manage (#1713)
```

Comment:
Also, could you add a note that a migration policy ``pi-update-policy-b9131d0686eb`` is added?

Reply:
done in  7d9d519c